api/src/main/java/org/pmiops/workbench/blockscore/BlockscoreService.java

```diff
@@ -1,15 +0,0 @@
-package org.pmiops.workbench.blockscore;
-
-import com.blockscore.models.Address;
-import com.blockscore.models.PaginatedResult;
-import com.blockscore.models.Person;
-
-/**
- * Encapsulate API for interfacing with Blockscore identity verification service.
- */
-public interface BlockscoreService {
-  public PaginatedResult<Person> listPeople();
-  public Person createPerson(
-    String firstName, String lastName, Address address, String dobString,
-    String documentType, String documentNumber);
-}
```

Comment:
When we do re-implement IDV, we should try to keep the service interface named cleanly, like `IdentityVerificationService`, etc. Vendor-specific naming, IMHO, is confusing.